cli/hzn.go

```diff
@@ -133,9 +134,9 @@ Environment Variables:
 	exNodeLong := exNodeListCmd.Flag("long", msgPrinter.Sprintf("When listing all of the nodes, show the entire resource of each node, instead of just the name.")).Short('l').Bool()
 	exNodeCreateCmd := exNodeCmd.Command("create", msgPrinter.Sprintf("Create the node resource in the Horizon Exchange."))
 	exNodeCreateNodeIdTok := exNodeCreateCmd.Flag("node-id-tok", msgPrinter.Sprintf("The Horizon Exchange node ID and token to be created. The node ID must be unique within the organization.")).Short('n').PlaceHolder("ID:TOK").String()
-	exNodeCreateNodeEmail := exNodeCreateCmd.Flag("email", msgPrinter.Sprintf("Your email address. Only needs to be specified if: the user specified in the -u flag does not exist, and you specified the 'public' org. If these things are true we will create the user and include this value as the email attribute.")).Short('e').String()
 	exNodeCreateNodeArch := exNodeCreateCmd.Flag("arch", msgPrinter.Sprintf("Your node architecture. If not specified, arch will leave blank.")).Short('a').String()
 	exNodeCreateNodeName := exNodeCreateCmd.Flag("name", msgPrinter.Sprintf("The name of your node")).Short('m').String()
+	exNodeCreateNodeType := exNodeCreateCmd.Flag("node-type", msgPrinter.Sprintf("The type of your node. The valid values are: device, cluster. If omitted, the default is device. However, the node type stay unchanged if the node already exists, only the node token will be updated.")).Short('T').String()
```

Comment:
You can specify the default value of the flag in the command definition, see line 331 default timeout flag for an example.

Reply:
The default is not 'device'. If the flag is missing and the node exists, the default is the same as the node type of the existing node.

Reply:
done